modules/service/data-service.js

```diff
@@ -367,11 +367,14 @@ class DataService {
             Operation_name: 'query_node',
             Id_operation,
         });
+        const quads = [];
+
         try {
             switch (type) {
-            // case 'SELECT':
-            //     result = this.implementation.execute(query);
-            //     break;
+            case 'SELECT':
+                result = await this.implementation.execute(query);
```

Comment:
I think we should change this to not directly ask but to go over triple store queue, like it is for construct. It is a small change one line here and 3 lines in handleTripleStoreRequest function